samples/queuesGettingStarted.ts

```diff
@@ -0,0 +1,78 @@
+import { OnMessage, OnError, MessagingError, delay, ServiceBusMessage, ReceiveMode, generateUuid, Namespace } from "../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+async function main(): Promise<void> {
+  await sendMessage();
+  await receiveMessage();
+}
+
+async function sendMessage(): Promise<void> {
+  let nsSend: Namespace;
+  nsSend = Namespace.createFromConnectionString(str);
+  const sendClient = nsSend.createQueueClient(path);
```

Comment:
You can combine the above 2 lines i.e declare and define at the same time.
Also, use `const` instead of `let` as we dont intend to change the value of this variable.

Reply:
Ok!